content/spin/kv-store-tutorial.md

```diff
@@ -1,14 +1,12 @@
-title = "Persistent Data: Spin"
+title = "Persistent Data Locally With Key Value Store"
 template = "spin_main"
 date = "2023-02-21T00:00:00Z"
 enable_shortcodes = true
 [extra]
-url = "https://github.com/fermyon/developer/blob/main//content/spin/kv-store.md"
+url = "https://github.com/fermyon/developer/blob/main//content/spin/kv-store-tutorial.md"
```

Comment:
New URL might need an update in the `sidebar.hbs` I think?  (and possibly a redirect in the site manifest)

Reply:
Good catch! I'll reach out to @karthik2804for help on this one.